distribution/docker/src/docker/Dockerfile

```diff
@@ -260,11 +257,24 @@ COPY ${bin_dir}/docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
 RUN chmod g=u /etc/passwd && \\
     chmod 0555 /usr/local/bin/docker-entrypoint.sh && \\
     find / -xdev -perm -4000 -exec chmod ug-s {} + && \\
-    ln -sf <%= docker_base == 'default' || docker_base == 'cloud' ? '/etc/ssl/certs/java/cacerts' : '/etc/pki/ca-trust/extracted/java/cacerts' %> \\
-      /usr/share/elasticsearch/jdk/lib/security/cacerts && \\
     chmod 0775 /usr/share/elasticsearch && \\
     chown elasticsearch bin config config/jvm.options.d data logs plugins
 
+<% if (docker_base == 'default' || docker_base == 'cloud') { %>
+# Update "cacerts" bundle to use Ubuntu's CA certificates (and make sure it
+# stays up-to-date with changes to Ubuntu's store)
+RUN set -eux ; \\
+    { \\
+      echo '#!/usr/bin/env bash'; \\
```

Comment:
Rather than `echo` out a script can we just keep this file in `docker/src` somewhere and `COPY` it into the image?

Reply:
@mark-vieira I extracted it to a script.